gui/src/2D/controls/multiLine.ts

```diff
@@ -62,7 +62,7 @@ export class MultiLine extends Control {
 
     /** Function called when a point is updated */
     public onPointUpdate = (): void => {
-        this._markAsDirty();
+        if (this.isVisible) this._markAsDirty();

```

Comment:
We need to add a MarkasDirty when is Visible will become true again then

Reply:
Let me fix that one

Reply:
I'm on it actually :)

Reply:
I will resolve it at control level

Reply:
You can remove it, it will be handle for all controls :)

Reply:
Okay

Reply:
Good call, it does make sense to handle this case for all controls :)